src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,43 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  handleCloseButton: () => void;
+  selectedUserId: number;
+  todo: Todo | null;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  handleCloseButton,
+  selectedUserId,
+  todo,
+}) => {
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [isUserLoaded, setIsUserLoaded] = useState(false);
+
+  const getUserFromServer = async () => {
+    try {
+      const user = await getUser(selectedUserId);
+
+      setSelectedUser(user);
+      setIsUserLoaded(true);
+    } catch (error) {
+      throw new Error('Something went wrong. Try again later, please');
```

Comment:
if you throw error here your App will crash , let us try to handle error without crash :)

Reply:
haven't fixed